Userland/Libraries/LibGfx/CharacterBitmap.h

```diff
@@ -9,26 +10,30 @@
 #include "Size.h"
 #include <AK/RefCounted.h>
 #include <AK/RefPtr.h>
+#include <AK/StringView.h>
 
 namespace Gfx {
 
-class CharacterBitmap : public RefCounted<CharacterBitmap> {
+class CharacterBitmap {
 public:
-    static NonnullRefPtr<CharacterBitmap> create_from_ascii(const char* asciiData, unsigned width, unsigned height);
-    ~CharacterBitmap();
+    CharacterBitmap() = delete;
+    constexpr CharacterBitmap(StringView ascii_data, unsigned width, unsigned height)
+        : m_bits { ascii_data }
+        , m_size { width, height }
```

Comment:
What do you mean? They are brackets. Do you prefer parentheses or are we having a cross-language difference in names of punctuation? :^)

Reply:
Is there a reason not to use `uniform initialization syntax`?

https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es23-prefer-the--initializer-syntax

Reply:
Changed as requested in Discord channel: 

Filmröllchen, Sampler of Audio — Today at 12:16 PM
@lenny re my bitching about m_member { initializer, blah, foo } is because there's probably not a single instance of this in the code base right now. If you like it more and have technical reasons, we can discuss a code style change, but your PR shouldn't be the place.